f2/src/i18n.config.js

```diff
@@ -11,7 +11,7 @@ export async function activate(locale) {
 
   try {
     catalog = await import(
-      /* webpackChunkName: "i18n-[index]" */ `@lingui/loader!./locales/${locale}.json`
+      /* webpackChunkName: "i18n-[index]" */ `./locales/${locale}.js`
```

Comment:
This should not merged in, it breaks the app - see deployment for this branch.

Reply:
Yes, you are right. This is an accident. I'll change it back. Thanks!